build_all.sh

```diff
@@ -20,12 +20,16 @@ if [ $? != 0 ] ; then usage ; fi
 
 eval set -- "$OPTS"
 
-JOBS="`cat /proc/cpuinfo | grep processor | wc -l`"
+# JOBS="`cat /proc/cpuinfo | grep processor | wc -l`"
+JOBS=4  # Core count
+
```

Comment:
Did the `cpuinfo` solution not work on mac? You could replace this with: `$([[ $(uname) = 'Darwin' ]] && sysctl -n hw.logicalcpu_max || lscpu -p | egrep -v '^#' | wc -l)` (from https://stackoverflow.com/a/23378780)

Reply:
Okay I'll add these, thanks!